examples/run/cpu/full_chain_algorithm.hpp

```diff
@@ -90,8 +97,10 @@ class full_chain_algorithm : public algorithm<track_state_container_types::host(
         const edm::silicon_cell_collection::host& cells) const override;
 
     private:
+    /// The host memory resource
+    vecmem::memory_resource& m_mr;
```

Comment:
You correctly removed the configuration objects. I think that's correct, it was just making things more obscure that they were kept in the class.

But then why add `m_mr`? The class doesn't need it. :confused: